### PR TITLE
Add post on Jupyter Book

### PR DESCRIPTION
**Pull request by astrojuanlu**
(Only missing the first paragraph, a bit tired now)

See https://github.com/executablebooks/jupyter-book/issues/1091 🥳 cc @choldgraf

### CODE REVIEW

**Comment by chrisjsewell** [comment]
Cheers @astrojuanlu

File: jupyter-book-read-the-docs.rst [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+   :tags: feature, jupyter
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      Jupyter Book is now supported on Read the Docs,
+      in this post we explain what is needed to make it work.
+
+Jupyter Book ❤️ Read the Docs
```
Comment (ericholscher) [inline jupyter-book-read-the-docs.rst:11]: Seems we'd swap this around for our post?

File: jupyter-book-read-the-docs.rst [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+the assumptions that Read the Docs makes to build the documentation of Sphinx projects don't hold:
+in particular, Jupyter Book uses a declarative configuration file ``_config.yml``
+that gets translated on the fly to the Sphinx dynamic configuration usually stored in ``conf.py``.
+As a result, Jupyter Book projects could not be hosted on Read the Docs - until now!
+
+.. _Sphinx: https://www.sphinx-doc.org/
+.. _MkDocs: https://www.mkdocs.org/
+
+How to do deploy a Jupyter Book project to Read the Docs
+--------------------------------------------------------
```
Comment (ericholscher) [inline jupyter-book-read-the-docs.rst:66]:
```suggestion
How to deploy a Jupyter Book project to Read the Docs
-----------------------------------------------------
```

**Review by ericholscher — APPROVED** [review]
This looks fine assuming the intro is written :) It doesn't have much information, but I think that's fine given that it's mostly just a "yay it works now" blog post.

We should definitely have some of this content in our docs, as well? The blog is good, but I'd hate for it to be the only place this information exists in our content.

File: jupyter-book-read-the-docs.rst [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      Jupyter Book is now supported on Read the Docs,
+      in this post we explain what is needed to make it work.
+
+Jupyter Book ❤️ Read the Docs
+=============================
+
```
Comment (ericholscher) [inline jupyter-book-read-the-docs.rst:13]: I'd also like to see a tl;dr here:

```suggestion
**We are proud to announce that Read the Docs now supports hosting documentation written in Jupyterbook.**
```

**Comment by astrojuanlu** [comment]
Yay, I'll write the intro tomorrow.

What extra information would you add? For more tutorial or guide-like content, we should add it to our docs I think

**Comment by ericholscher** [comment]
> What extra information would you add? For more tutorial or guide-like content, we should add it to our docs I think

I don't think we need much more than this content, but mostly just want a search to return...something relevant: https://docs.readthedocs.io/en/latest/?rtd_search=jupyter%20book

Perhaps just a section in an existing doc noting that we support it, to start?

**Comment by choldgraf** [comment]
(just as an aside, this reminds me of a conversation I'd had with some Turing Way folks, who mentioned that it would be really helpful if Sphinx let you put search term metadata in a document that would then boost that page in search results that used one of those terms...from this conversation I'm guessing that this isn't possible, but maybe you would find this useful too?)

**Comment by astrojuanlu** [comment]
Companion documentation addition: https://github.com/readthedocs/readthedocs.org/pull/8669

File: jupyter-book-read-the-docs.rst (hunk 1 above)
Comment (astrojuanlu) [reply jupyter-book-read-the-docs.rst:11]: I see the marketing reasoning behind it, although I'll try to hint in the text that JB did all the work :)

**Comment by ericholscher** [comment]
> (just as an aside, this reminds me of a conversation I'd had with some Turing Way folks, who mentioned that it would be really helpful if Sphinx let you put search term metadata in a document that would then boost that page in search results that used one of those terms...from this conversation I'm guessing that this isn't possible, but maybe you would find this useful too?)

@choldgraf I know we've talked about this in the past, but I asked Santos on our team and we ended up with this issue: ￼https://github.com/readthedocs/readthedocs.org/issues/8670

File: jupyter-book-read-the-docs.rst (hunk 3 above)
Comment (astrojuanlu) [reply jupyter-book-read-the-docs.rst:13]: Done!

**Comment by astrojuanlu** [comment]
🤖 Rendered version: https://readthedocs-blog--141.org.readthedocs.build/jupyter-book-read-the-docs/

**Comment by astrojuanlu** [comment]
Ready for re-review, only added the intro

File: jupyter-book-read-the-docs.rst [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+We are proud to announce that now Jupyter Book projects are supported on Read the Docs!
+
+Both Read the Docs and The Executable Book Project, the folks behind Jupyter Book,
+share a common passion for documentation,
+and we have been collaborating on various topics for some time already.
+For example, we started promoting MyST in favor of our recommonmark
+:doc:`back in April this year </newsletter-april-2021>`,
+and we wrote a guide on :doc:`using Jupyter notebook with Sphinx <readthedocs:guides/jupyter>`
+that benefitted a lot from their feedback.
```
Comment (ericholscher) [inline jupyter-book-read-the-docs.rst:22]: Great summary. :heart:

**Review by ericholscher — APPROVED** [review]
Ship it!

**Comment by choldgraf** [comment]
:shipit:

**Pull request merged**